packages/@expo/cli/ts-declarations/freeport-async/index.d.ts

```diff
@@ -3,6 +3,6 @@ declare module 'freeport-async' {
     hostnames?: Array<string | null>;
   }
 
-  function freePortAsync(rangeStart: number, options?: FreePortOptions): Promise<number>;
+  function freePortAsync(rangeStart?: number, options?: FreePortOptions): Promise<number>;
```

Comment:
Going to skip this, I used this in the E2E tests - but `freeport-async` is quite terrible to "find an available port". It will always return the same port (`11000`) for some reason.